packages/connect-voting/src/__test__/votes.test.ts

```diff
@@ -1,11 +1,19 @@
+import { BigNumber } from 'ethers'
+import { App, connect } from '@aragon/connect'
 import { VotingConnectorTheGraph, Vote, Cast } from '../../src'
+import { Action, VoteStatus } from '../types'
 
 const VOTING_SUBGRAPH_URL =
-  'https://api.thegraph.com/subgraphs/name/aragon/aragon-voting-rinkeby-staging'
+  'https://api.thegraph.com/subgraphs/name/aragon/aragon-voting-rinkeby'
```

Comment:
the tests did not ran successfully, even after swapping the VOTING_SUBGRAPH_URL as stated in the PR description.

- can you provide more context or instruction how you run test locally?
- Is there a way to run the test successfully without swapping the URL?